e2e/benchmarks/local-benchmark/index.html

```diff
@@ -93,6 +93,7 @@ <h2>TensorFlow.js Model Benchmark</h2>
   <script src="https://unpkg.com/@tensorflow/tfjs-converter@latest/dist/tf-converter.js"></script>
   <script src="https://unpkg.com/@tensorflow/tfjs-backend-wasm@latest/dist/tf-backend-wasm.js"></script>
   <script src="https://unpkg.com/@tensorflow/tfjs-automl@latest/dist/tf-automl.js"></script>
+  <script src="https://unpkg.com/@tensorflow/tfjs-backend-webgpu@latest/dist/tf-backend-webgpu.min.js"></script>
```

Comment:
Only add it when webgpu backend is supported. You can add it like below in the script body:
```
    if (isBackendSupported('webgpu'))
    {
      var script = document.createElement("script");
      script.setAttribute("src", "https://unpkg.com/@tensorflow/tfjs-backend-webgpu@latest/dist/tf-backend-webgpu.min.js");
      document.getElementsByTagName("body")[0].appendChild(script);
    }

Reply:
I think simply dynamically add a script tag doesn't gurantee that user can use the tag immediately, because 
chrome needs extra time to make this script ready. So we need some logic done here: https://github.com/tensorflow/tfjs/pull/5094